include/sys/multi_heap.h

```diff
@@ -51,10 +51,16 @@ struct sys_multi_heap;
 typedef void *(*sys_multi_heap_fn_t)(struct sys_multi_heap *mheap, void *cfg,
 				     size_t align, size_t size);
 
```

Comment:
Looks like you need to fix the formatting for you commit message. Please add a new line between subject and body. Also, please try to trim the commit message line length at around 100 chars.